client/src/modules/events/pages/eventsPage.tsx

```diff
@@ -71,11 +76,25 @@ export const EventsPage: NextPage = () => {
 
   const isLoading = loading || !data;
   if (isLoading || error) return <Loading loading={isLoading} error={error} />;
+  const canAuthenticateWithGoogle = useCheckPermission(
+    Permission.GoogleAuthenticate,
+  );
```

Comment:
This can't be after a conditional, since every hook has to be called on every render: [rules of hooks](https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level)